packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -5,7 +5,13 @@ category: rate limits
 
 # System Log events for rate limits
 
-Okta offers three standard System Log event types that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations.
+## System Log event types
+
+Okta offers specific [standard System Log events](/docs/reference/rl-system-log-events/#standard-system-log-event-types), [org-based System Log events](/docs/reference/rl-system-log-events/#org-based-system-log-event-types) and [client-based System Log events](/docs/reference/rl-system-log-events/#client-based-system-log-event-types).
+
+### Standard System Log event types
+
+There are three standard System Log event types that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations:
 
 * `system.operation.rate_limit.violation`<br>
```

Comment:
can we also link each of these event types to their listing in the event catalog? e.g. https://developer.okta.com/docs/reference/api/event-types/?q=system.operation.rate_limit.violation

Reply:
A note that this comment was implemented except for the last one `system.operation.rate_limit.notification`. This info was provide by Peter in a Slack conversation:
"technically that event type doesn’t exist yet
but it may/will be introduced in the future
i think we can leave the link out for now"